messaging/src/main/java/org/axonframework/eventhandling/EventHandler.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2010-2025. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.eventhandling;
+
+import jakarta.annotation.Nonnull;
+import org.axonframework.messaging.MessageStream;
+import org.axonframework.messaging.configuration.MessageHandler;
+import org.axonframework.messaging.configuration.NoMessage;
+import org.axonframework.messaging.unitofwork.ProcessingContext;
+
+/**
+ * Interface describing a handler of {@link EventMessage events}.
+ *
+ * @author Allard Buijze
+ * @author Steven van Beelen
+ * @since 0.1
```

Comment:
5.0.0 right? :)

Reply:
Well, technically this is a replacement for the `EventMessageHandler` -> https://github.com/AxonFramework/AxonFramework/blob/master/messaging/src/main/java/org/axonframework/eventhandling/EventMessageHandler.java
Hence why I took the since tag from that class for this one.

Reply:
Ahaaa, makes sense